#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-fop-config.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: The default configuration of op-panel is just useless. Lets fix it 
## DP: before it gets into the package:
## DP:   - if no web_hostname is set, it will be automatically detected
## DP:   - flash_dir should be the place on the HD. it looks for variables.txt in that dir
## DP:     TODO: should be modified to "/var/lib/op-panel/"
## DP:   - use a sane default password for the UI
## DP:   - use a default passwords for the manager


@DPATCH@
diff -urNad freepbx-2.1.1/amp_conf/htdocs_panel/op_server.cfg /tmp/dpep.U4TyWY/freepbx-2.1.1/amp_conf/htdocs_panel/op_server.cfg
--- freepbx-2.1.1/amp_conf/htdocs_panel/op_server.cfg	2006-05-13 09:01:22.000000000 +0300
+++ /tmp/dpep.U4TyWY/freepbx-2.1.1/amp_conf/htdocs_panel/op_server.cfg	2006-08-16 17:12:42.000000000 +0300
@@ -3,8 +3,8 @@
 manager_host=127.0.0.1      
 
 ; user and secret for connecting to * manager
-manager_user=user
-manager_secret=secret
+manager_user=op-panel
+manager_secret=op_panel_password
 
 ; The optional event_mask for filtering manager events.
 ; Asterisk will send only the events you request
@@ -31,14 +31,14 @@
 ; the flash movie resides (just the hostname, without directories)
 ; This value might be omited. In that case the flash movie will
 ; try to connect to the same host as the web page.
-web_hostname=www.myexample.com
+web_hostname=
 
 ; location of the .swf file in your disk (must reside somewhere 
 ; inside your web root)
-flash_dir=/var/www/html/panel
+flash_dir=/usr/share/op-panel/flash/
 
 ; secret code for performing hangups and transfers
-security_code=dkd4393kld
+security_code=password
 
 ; Frequency in second to poll for sip and iax status
 poll_interval=120
@@ -170,7 +170,7 @@
 ; extension@context, it will originate a call to
 ; voicemailmain when double clicking on the MWI icon
 ; for any button. 
-voicemail_extension = 3000@features
+;# voicemail_extension = 3000@features
 
 ; You can have panel contexts with their own
 ; button layout and configuration. The following entry
